package.json

```diff
@@ -10,7 +10,7 @@
     "dist": "mkdirp dist/ && npm run browserify -s -- --debug | exorcist dist/aframe.js.map > dist/aframe.js && uglifyjs dist/aframe.js -c warnings=false -m -o dist/aframe.min.js",
     "gh-pages": "npm run ghpages",
     "ghpages": "node ./scripts/gh-pages",
-    "lint": "semistandard -v $(git ls-files '*.js') | standard-reporter --stylish",
+    "lint": "semistandard -v $(git ls-files '*.js' | grep -v src/lib/) | standard-reporter --stylish",
```

Comment:
@ngokevin just change [this line](https://github.com/ngokevin/aframe/blob/stylesrc/package.json#L86) to be `src/lib/…`